docs/tutorials/quickstart.md

```diff
@@ -105,9 +92,9 @@ To use opentime without opentimelineio, link with -lopentime instead, and compil
 
 ### Linux / GDB / LLDB
 
-From your virtual environment, compile with debug flags:
+From your virtual environment, compile with debug flags (also works on Windows):
```

Comment:
the syntax `env foo=1 python` doesn't actually work on windows

Reply:
What do you think if I go with:

> To compile in debug mode, set the `OTIO_CXX_DEBUG_BUILD` environment variable to any value and then `python -m pip install .`

?

Reply:
Sure!